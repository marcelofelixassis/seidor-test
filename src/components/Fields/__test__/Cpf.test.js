import React from 'react';
import { shallow } from 'enzyme';
import Cpf from '../cpf';
test('renders the component', () => {
  const component = shallow(<Cpf  disabled={false} name={"teste"} value={"123"} onChange={jest.fn()}/>);
  expect(component).toMatchSnapshot();
});