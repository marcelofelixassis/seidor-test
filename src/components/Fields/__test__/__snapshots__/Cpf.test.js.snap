// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component 1`] = `
<InputElement
  disabled={false}
  mask="999.999.999-99"
  maskChar={false}
  name="teste"
  onChange={[MockFunction]}
  value="123"
/>
`;
